pravega-video-server/src/main.rs

```diff
@@ -469,12 +470,30 @@ mod models {
             self,
             scope_name: String,
         ) -> anyhow::Result<ListStreamsResult> {
+            use pravega_controller_client::paginator::list_streams;
+            use futures::future;
+            let ss = scope_name.clone();
```

Comment:
Get rid of this line. Just be sure to use `scope_name.clone()` everywhere so nobody takes ownership of `scope_name`.